### PR TITLE
Enhancement: Ratings UI Improved

### PR DESCRIPTION
**Pull request by mishal23**
# Status
Ready

# Description
The rating UI is improved and provides more functionality and control on the web admin to decide who would rate the extension (for eg. only once a user can rate, only authenticated users can rate)

# ToDos
- [x] Python 2&3 Support
- [x] PEP-8
- [x] Master branch up-to-date
- [x] Tested Locally

# Steps to Test or Reproduce
Run the Django App locally, create necessary apps, and then try to rate the app

# Deploy Notes
Ensure `django-star-ratings` is installed on the production server.

### CODE REVIEW

**Comment by abhijithanilkumar** [comment]
Can you add a screenshot for this? Did you make the changes suggested during the last call?

File: src/download/apps.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class DownloadConfig(AppConfig):
```
Comment (abhijithanilkumar) [inline src/download/apps.py]: What is this for?

File: src/appstore/settings/base.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 # from django.core.urlresolvers import reverse_lazy
 from django.urls import reverse_lazy
 from os.path import dirname, join, exists
+import rest_framework
```
Comment (abhijithanilkumar) [inline src/appstore/settings/base.py]: Do we need this?

File: src/backend/urls.py [hunk 3]
```diff
@@ -75,8 +75,4 @@
         'api/search/',
         views_api.SearchApiViewSet.as_view({'post': 'create'}),
         name='search'),
-    path(
```
Comment (abhijithanilkumar) [inline src/backend/urls.py]: Intentional?

**Comment by mishal23** [comment]
Cumulative merged in #59

**Pull request closed without merging**